doc/api/cli.md

```diff
@@ -1370,10 +1359,10 @@ added: v12.0.0
 
 This configures Node.js to interpret `--eval` or `STDIN` input as CommonJS or
 as an ES module. Valid values are `"commonjs"`, `"module"`, `"module-typescript"` and `"commonjs-typescript"`.
-The `"-typescript"` values are available only in combination with the flag `--experimental-strip-types`.
+The `"-typescript"` values not available with the flag `--no-experimental-strip-types`.
```

Comment:
nit: this grammar does not read well with the new negation. other areas reviewed so far were fine

```suggestion
The `"-typescript"` values are not available with the flag `--no-experimental-strip-types`.
```

Reply:
The grammar here needs correction. Was there something wrong with the above suggestion?

Reply:
I thought I applied the suggestion🤔

Reply:
applied